### ISSUE TITLE
Builder tasks block health check loop

### ISSUE THREAD

**Issue by ARMmaster17** [ARMmaster17/Captain#95]
As of right now in v0.1.0, although builder tasks run in parallel (under most circumstances), the health check loop waits for the builder tasks to finish before continuing to other flights and airspaces. Two possible solutions:

Short term solution: After a builder job has been kicked off, continue the health checks. Store the plane state in the state database. Consider any plane being built as being in an OK state. If the builder fails, delete the plane on the next health check. Any plane with the status DELETING can be considered dead.

Long term solution: Build a cron system that kicks off health checks for specific formations at specified intervals. Have a work queue and scaled out health checker threads. This will have to be a separate issue as Captain will need to be much more distributed than it currently is.
